src/transformers.ts

```diff
@@ -0,0 +1,17 @@
+import { PanelData } from '@grafana/data';
+
+export const toPerspectiveData = ({ series }: PanelData) => {
```

Comment:
I would love to see if [DataFrameView](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/dataframe/DataFrameView.ts) could acomplish the same thing without actually creating new objects for each row.

Also... knowing that they are using arrow under the hood... can we pass them an arrow table?  [maybe?](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/dataframe/ArrowDataFrame.ts#L114)

Reply:
I don't see how else it could work when it basically converts:
```js
{
  series: [{
    fields: [
      { name: 'AA', values: [0, 1, 2, 3] },
      { name: 'BB', values: [4, 5, 6, 7] }
    ]
  }]
}
```
to:
```js
[
  { AA: 0, BB: 4 },
  { AA: 1, BB: 5 },
  { AA: 2, BB: 6 },
  { AA: 3, BB: 7 }
]
```

Reply:
I can't `import` `ArrowDataFrame` because it is both not `export`'ed in 6.6, and all `src/**/*.ts` files are not included in the published package.